tox.ini

```diff
@@ -0,0 +1,73 @@
+[tox]
+envlist = bandit,black,isort,flake8,mypy,python3.9,python3.10
+
+[gh-actions]
+python =
+    3.9: python3.9
+    3.10: python3.10
+
+[testenv]
+deps =
+    -rrequirements.txt
+    -rrequirements-dev.txt
+whitelist_externals =
+    make
+    mkdir
+    rm
+passenv = TOX_ENV_DIR
+usedevelop = true
+
+commands =
+    pytest \
+        --cov-config .coveragerc --cov=pyarn --cov-report term \
+        --cov-report xml --cov-report html
+
+[testenv:black]
+description = black checks [Mandatory]
+skip_install = true
+deps =
+    # Pin the version of black and click to avoid a newer version causing tox to fail
+    black==22.6.0
+    click==8.0.3
+commands =
+    black --check --diff pyarn tests
+
+[testenv:isort]
+skip_install = true
+deps =
+    isort[colors]
+commands =
+    isort --check --diff --color pyarn tests
+
+[testenv:flake8]
+skip_install = true
+deps =
+    flake8==3.9.2
+    flake8-docstrings==1.6.0
+commands =
+    flake8
+
+[flake8]
+show-source = True
+max-line-length = 100
```

Comment:
We should configure black and isort to use line length 100 as well, or use black's default 88 for all three

Reply:
And we might want to use `profile = "black"` for isort https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

Reply:
I might be blind, but I don't see line length configured anywhere :smile: